### ISSUE TITLE
Is flipY() exported?

### ISSUE THREAD

**Issue by tunneling** [chrvadala/transformation-matrix#90]
I'm a little rusty on matrix math but I think this is the right library. I'm trying to do coordinate conversions from one system to another. This is for a project where I'm drawing on Canvas and using some json Objects for points.

For Canvas; the origin is at the top left.
For the json Objects data; the origin is at the bottom left.

I was thinking I needed to simply flip the Y axis to convert from the json Objects to draw it on Canvas. However, I get the following error:

```
import { translate, applyToPoint, flipY } from 'transformation-matrix';
                                  ^^^^^
SyntaxError: Named export 'flipY' not found. The requested module 'transformation-matrix' is a CommonJS module, which may not support all module.exports as named exports.
```

My need is as follows:
```
let jsonPointOrigin = {x: 0, y: 0};
let canvasPointBottomLeft = applyToPoint(matrix, jsonPointOrigin);
let jsonPointMax = {x: 407, y: 407};
let canvasPointTopRight = applyToPoint(matrix, jsonPointMax );

Output:
{x: 0, y: 407}
{x: 407, y: 0}

```

**Comment by chrvadala** [comment]
I can confirm that in the current version flipX, flipY and flipOrigin aren't exported. I made this commit https://github.com/chrvadala/transformation-matrix/commit/8c4506c33b70332f119dc2948de9c3446a3364c5 that fixes this issue.
I will release a new version soon.

Thanks for letting me know

**Comment by chrvadala** [comment]
Fixed with 2.11.1
